### PR TITLE
feat: GitHub Actions support

### PR DESCRIPTION
**Pull request by gr2m**
Fixes https://github.com/semantic-release/github/issues/182. For the time being, you can install my fork

```
npm install -D @gr2m/semantic-release-github
```

### CODE REVIEW

File: lib/verify.js [hunk 1]
```diff
@@ -52,6 +53,12 @@ module.exports = async (pluginConfig, context) => {
   } else if (githubToken && !errors.find(({code}) => code === 'EINVALIDPROXY')) {
     const github = getClient({githubToken, githubUrl, githubApiPathPrefix, proxy});
 
+    // https://github.com/semantic-release/github/issues/182
+    if (githubToken.substr(0, 3) === 'v1.' && env.GITHUB_ACTION) {
```
Comment (pvdlg) [inline lib/verify.js]: I would rather avoid to depend on the format of the token. So I would only check for `env.GITHUB_ACTION`.

Comment (gr2m) [reply lib/verify.js]: I’ve also added a longer comment explaining the early return in case `env.GITHUB_ACTION` is truthy

**Comment by KengoTODA** [comment]
This feature doesn't work as expected [in my workflow run](https://github.com/KengoTODA/gradle-boilerplate/runs/232820570#step:7:48), I will investigate and share knowledge if possible.

**Comment by gr2m** [comment]
Thanks for looking into it, we'd appreciate if you could see what went wrong there. One difference I see from my own setup is that you use yarn while I use npm. But I don't think there should be any difference in how environment variables are passed?

**Comment by KengoTODA** [comment]
I found that it's caused by my `yarn.lock`.

`semantic-release v15.13.24` depends on `^@semantic-release/github v5.1.0` and it overwrote the dependency on `v5.4.4`. So semantic-release `v16.0.0` release will solve this potential issue.

Thanks for your help!

**Pull request merged**